slang-gfx.h

```diff
@@ -909,6 +909,25 @@ struct ComputePipelineStateDesc
     IShaderProgram*  program;
 };
 
+struct RayTracingPipelineFlags
```

Comment:
Elsewhere we've done this as 

```
typedef uint32_t RayTracingPipelineFlags;
struct RayTracingPipelineFlag
{
    enum Enum : RayTracingPipelineFlags
    {
        None = 0,
        SkipTriangles = 1,
        SkipProcedurals = 2,
    };
};
```
Such that flags can be easily combined, or a single flag specified.
An issue is that 'Flags' then doesn't have type safety.